### ISSUE TITLE
having noSmooth(); in steup will trigger a duplicate settings() warning

### ISSUE THREAD

**Issue by clankill3r** [benfry/processing4#823]
Processing 4.3
Windows 10

I thought I was going crazy :)

```java
void settings() {
 
}

void setup() {
  noSmooth();
}
```
![Screenshot 2024-02-18 223447](https://github.com/benfry/processing4/assets/738650/e2b217a8-588e-471b-98e4-97b83c18780e)

**Comment by JuzerNejm10** [comment]
Yeah, happened to me a couple of times with noSmooth() or smooth(). That must go into settings() . 
But, on the other hand, settings() are very restrictive, you cannot have eg. frameRate(), background(), stroke(), fill(), noCursor(), surface.setVisible() etc. in there - you'll get NullPointerException.
